monorepo-folders/package.json

```diff
@@ -17,6 +17,13 @@
     "concurrently": "^6.3.0"
   },
   "resolutions": {
+    "@temporalio/worker": "1.4",
+    "@temporalio/workflow": "1.4.3",
+    "@temporalio/client": "^1.0.0",
+    "@temporalio/activity": "^1.0.0",
+    "@temporalio/common": "^1.0.0",
+    "@temporalio/internal-workflow-common": "^1.0.0",
+    "@temporalio/proto": "^1.0.0",
```

Comment:
I'm approving this PR already but please provide clarification why this is required in the first place.